### PR TITLE
Prepare release 1.1 for location-verification + README + CHANGELOG

### PR DESCRIPTION
**Pull request by jlurien**
#### What type of PR is this?

* correction
* subproject management

#### What this PR does / why we need it:

Prepare location-verification for r1.1
Some fixes after review
Update README (for the 3 APIs)
Update CHANGELOG (for location-verification + placeholders for the others)

### CODE REVIEW

**Comment by jlurien** [comment]
Before creating the r1.1 release we have to:
- Merge the pending PRs for geofencing (maybe with the exception of the test plan)
- Create a PR for geofencing adjusting versions for r1.1
- Merge the PR for location-retrieval for r1.1
- Merge this PR for location-verification + README/CHANGELOG for r1.1

File: code/API_definitions/location-verification.yaml [hunk 1]
```diff
@@ -80,25 +80,23 @@ info:
...
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.3.0-wip
+  x-camara-commonalities: 0.4.0
 externalDocs:
   description: Product documentation at CAMARA
   url: https://github.com/camaraproject/
 servers:
-  - url: "{apiRoot}/location-verification/v0"
+  - url: "{apiRoot}/location-verification/v1"
```
Comment (bigludo7) [inline code/API_definitions/location-verification.yaml:92]: Should be - url: "{apiRoot}/location-verification/v1rc1" ?

@hdamker could you confirm?

Comment (jlurien) [reply code/API_definitions/location-verification.yaml:92]: Yes, no doubt, I'll correct it

Comment (jlurien) [inline code/API_definitions/location-verification.yaml:92]:
```suggestion
  - url: "{apiRoot}/location-verification/v1rc1"
```

File: code/Test_definitions/location-verification.feature [hunk 2]
```diff
@@ -12,7 +12,7 @@ Feature: CAMARA Device location verification API, vwip - Operation verifyLocatio
   # References to OAS spec schemas refer to schemas specifies in location-verification.yaml, version 0.2.0
 
   Background: Common verifyLocation setup
-    Given the resource "/location-verification/v0/verify"                                                              |
+    Given the resource "/location-verification/v1/verify"                                                              |
```
Comment (jlurien) [inline code/Test_definitions/location-verification.feature:15]:
```suggestion
    Given the resource "/location-verification/v1rc1/verify"                                                              |

```

**Review by bigludo7 — APPROVED** [review]
LGTM :)

**Pull request merged**